src/dispatchers/mouseDispatcher.ts

```diff
@@ -2,110 +2,119 @@
 
 module Plottable {
 export module Dispatcher {
-  export class Mouse extends Dispatcher.AbstractDispatcher {
-    private _mouseover: (location: Point) => any;
-    private _mousemove: (location: Point) => any;
-    private _mouseout: (location: Point) => any;
+  export class Mouse {
+    private static dispatcherKey = "__Plottable_Dispatcher_Mouse";
```

Comment:
Double-underscore and named because it's being set as a property on a DOM node, so I want to minimize changes of collision. D3 uses `__data__` for example.